### PR TITLE
Twitter posts that don't have text dont show up in popular anymore

### PR DESCRIPTION
**Pull request by m-hakkestad**
Can be merged. Tweets where the text is "" aka empty it will not render

### CODE REVIEW

File: src/pages/Popular.js [hunk 1]
```diff
@@ -26,22 +26,27 @@ class Popular extends React.Component {
...
+            let filteredContent = this.state.data.filter(content =>
+                    this.state.filters.includes(content.platform.toLowerCase())
+            )
             feedContent = filteredContent.map(curContent => {
-                return <PopularComponent
-                    key={curContent.postid}
-                    data={curContent.platformcontent}
-                    userId={this.props.userId}
-                    userFollowing={curContent.usrfollowinginfluencer} />
+                console.log(curContent)
```
Comment (codebyjonas) [inline src/pages/Popular.js]: Please remove the `console.log()`

**Comment by codebyjonas** [comment]
Please provide more information about this issue, in my feed there is text in the twitter component.

**Comment by m-hakkestad** [comment]
Tweets with links in the beginning of messages don't have any text when returned from aggregate content. Which leaves the text field empty. This checks if the text field is empty and if it is does not render them in the popular feed. I will add this fix to the other pages tomorrow and remove the console.log

**Comment by codebyjonas** [comment]
> Tweets with links in the beginning of messages don't have any text when returned from aggregate content. Which leaves the text field empty. This checks if the text field is empty and if it is does not render them in the popular feed. I will add this fix to the other pages tomorrow and remove the console.log

Great!

File: src/components/popular/PopularComponent.js [hunk 2]
```diff
@@ -41,7 +41,8 @@ class PopularComponent extends React.Component {
     }
 
     render() {
-        if (this.props.data != null) {
+        if (this.props.data != null && this.props.data.tweetText != "") {
+            console.log(this.props.data);
```
Comment (codebyjonas) [inline src/components/popular/PopularComponent.js]: Please remove

File: src/pages/Popular.js [hunk 3]
```diff
@@ -26,16 +26,18 @@ class Popular extends React.Component {
                 'Accept': 'application/json, text/plain, */*',
                 'Content-Type': 'application/json'
             },
-            body: JSON.stringify({ assetType: ['all'], filterType: ['popular'], filterValue: [this.props.userId], limit: 100 })
+            body: JSON.stringify({ assetType: ['all'], filterType: ['popular'], filterValue: [this.props.userId,0], limit: 100 })
```
Comment (codebyjonas) [inline src/pages/Popular.js]: Use the props.adminId instead of 0

**Comment by codebyjonas** [comment]
Please fix conflictin files.

**Comment by m-hakkestad** [comment]
Conflicts solved

**Pull request merged**